src/components/interface/EcosystemHeader/EcosystemHeader.tsx

```diff
@@ -0,0 +1,75 @@
+import { SystemStyleObject, Theme, Box } from "@mui/system";
+import { Panel, Typography } from "src/components/atomic";
+
+export type EcosystemHeaderProps = {
+    lpOptimizerTag: string;
+    lpOptimizerCount: number;
+    alphaVaultTag: string;
+    alphaVaultCount: number;
+};
+
+const EcosystemHeader = ({lpOptimizerTag, lpOptimizerCount, alphaVaultTag, alphaVaultCount}: EcosystemHeaderProps) => {
+  
+    const titleStyles: SystemStyleObject<Theme> = { 
+        fontSize: '40px', 
+        lineHeight: '1.2', 
+        color: '#E5E1F9',
+        fontWeight: '700',
+    };
+
+    const copyStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+        color: '#9B97AD', 
+        marginTop: "8px",
+    }
+    
+    const tagStyles: SystemStyleObject<Theme> = {
+        fontSize: '14px',
+    };
+
+    const boxStyles: SystemStyleObject<Theme> = {
+        display: "flex",
+        borderRadius: "4px",
+        padding: "0px 8px",
+    }
+      
+    const renderContent = () => {
+        return (
+            <Panel sx={{padding: 0, background: 'transparent'}}>
+                <Box>
+                    <Typography variant='h1' sx={titleStyles}>
+                        VOLTZ ECOSYSTEM
+                    </Typography>
+                    <Typography variant='body1' sx={copyStyles}>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed nec sit iaculis cras elit dictum massa. Sit metus amet, tincidunt odio. Tristique sagittis, nisl in eu eu vestibulum et. Ut sed mauris urna justo, dictumst molestie posuere.
+                    </Typography>
+                </Box>
+                <Box sx={{marginTop: "16px", display: "flex"}}>
+                    <Box sx={{...boxStyles, border: "1px solid #FF4AA9"}}>
```

Comment:
don't we have these colors in our pallete already? This seems as wildStrawbery.base

Reply:
Let me do this later.